src/components/profile/user-profile.js

```diff
@@ -166,7 +166,9 @@ class UserItemView extends Component {
               <Tab
                 label="Future events"
                 icon={
-                  <IconButton color={this.state.value === 0 ? "" : "primary"}>
+                  <IconButton
+                    color={this.state.value === 0 ? "default" : "primary"}
```

Comment:
As well simplify this condition

Reply:
if I change same examinations in file functional of like and dislike don`t work correct

Reply:
Ok, thanks for verification